pkg/command/executor.go

```diff
@@ -61,8 +58,17 @@ func (c *command) Exec() (out []byte, err error) {
 		}
 	}()
 
-	clog.Debug("execute", "command", cmd.Args, "dir", cmd.Dir)
-	if c.ioConnected {
+	out, err = e.exec(ctx, c, cmd)
+
+	signal.Reset()
+	close(sigCh)
+
+	wg.Wait()
+	return
+}
+
+func (e *executor) exec(ctx context.Context, c *Command, cmd *exec.Cmd) (out []byte, err error) {
```

Comment:
**[unparam]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
(*executor).exec - ctx is unused

Reply:
fixed in 1e9fd43